app/scripts/controllers/visualizer.js

```diff
@@ -2,18 +2,21 @@
 /* global moment: false */
```

Comment:
There weren't any existing tests for this, however I added one in. I had issues running npm test in vagrant though (clean vagrant, fresh node_modules install etc.), must be due to new package versions. I've gone ahead and upgraded all dev-deps. The new jshint is enforcing a new rule though so there are quite a few changes stemming there from.